src/manager.js

```diff
@@ -13,24 +13,102 @@ import './images/turing-logo.png'
 
 //date
 let manager = new Manager();
+$('#future-bookings').hide();
 
-$('.date').html(manager.todaysDate());
 
 let bookingData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/bookings/bookings')
   .then(data => data.json())
   .then(data => data.bookings)
 
-
 let roomData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/rooms/rooms')
   .then(data => data.json())
   .then(data => data.rooms)
 
-Promise.all([bookingData, roomData]).then((requiredData) => {
-  manager.rooms = requiredData[1];
+let userData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/users/users')
+  .then(data => data.json())
+  .then(data => data.users)
+
+Promise.all([bookingData, roomData, userData]).then((requiredData) => {
   manager.booked = requiredData[0];
-  let bookedTodayData = manager.bookings();
-  $('.revenue').html(manager.totalSpend(bookedTodayData));
+  manager.rooms = requiredData[1];
+  manager.users = requiredData[2];
+  let bookedTodayData = manager.bookings(manager.todaysDate());
+  $('.date').html(manager.todaysDate());
+  $('.revenue').html(manager.getTotalSpend(bookedTodayData));
   $('.percentage').html((bookedTodayData.length/manager.rooms.length) * 100)
```

Comment:
I like how you calculated the percentage here.   I have a method in my tapeChart class that calculates this but I think its simple enough that having it in this js file is fine.  I can see where that decision making gets easy to double think.